src/drt/src/pa/FlexPA_prep.cpp

```diff
@@ -404,322 +409,302 @@ bool FlexPA::enclosesOnTrackPlanarAccess(
+                            const frLayerNum layer_num,
+                            const bool allow_planar,
+                            const bool allow_via,
+                            frAccessPointEnum lower_type,
+                            const frAccessPointEnum upper_type,
+                            const bool is_macro_cell_pin)
 {
-  auto layer = getDesign()->getTech()->getLayer(layerNum);
-  const auto minWidthLayer1 = layer->getMinWidth();
+  auto layer = getDesign()->getTech()->getLayer(layer_num);
+  const auto min_width_layer1 = layer->getMinWidth();
   if (std::min(gtl::delta(rect, gtl::HORIZONTAL),
                gtl::delta(rect, gtl::VERTICAL))
-      < minWidthLayer1) {
+      < min_width_layer1) {
     return;
   }
-  frLayerNum secondLayerNum = 0;
-  if (layerNum + 2 <= getDesign()->getTech()->getTopLayerNum()) {
-    secondLayerNum = layerNum + 2;
-  } else if (layerNum - 2 >= getDesign()->getTech()->getBottomLayerNum()) {
-    secondLayerNum = layerNum - 2;
+  frLayerNum second_layer_num = 0;
+  if (layer_num + 2 <= getDesign()->getTech()->getTopLayerNum()) {
+    second_layer_num = layer_num + 2;
+  } else if (layer_num - 2 >= getDesign()->getTech()->getBottomLayerNum()) {
+    second_layer_num = layer_num - 2;
   } else {
-    logger_->error(
-        DRT, 68, "prepPoint_pin_genPoints_rect cannot find secondLayerNum.");
+    logger_->error(DRT, 68, "genAPsFromRect cannot find second_layer_num.");
   }
-  const auto minWidthLayer2
-      = getDesign()->getTech()->getLayer(secondLayerNum)->getMinWidth();
-  auto& layer1TrackCoords = trackCoords_[layerNum];
-  auto& layer2TrackCoords = trackCoords_[secondLayerNum];
-  const bool isLayer1Horz = (layer->getDir() == dbTechLayerDir::HORIZONTAL);
+  const auto min_width_layer2
+      = getDesign()->getTech()->getLayer(second_layer_num)->getMinWidth();
+  auto& layer1_track_coords = track_coords_[layer_num];
+  auto& layer2_track_coords = track_coords_[second_layer_num];
+  const bool is_layer1_horz = (layer->getDir() == dbTechLayerDir::HORIZONTAL);
 
-  std::map<frCoord, frAccessPointEnum> xCoords;
-  std::map<frCoord, frAccessPointEnum> yCoords;
+  std::map<frCoord, frAccessPointEnum> x_coords;
+  std::map<frCoord, frAccessPointEnum> y_coords;
   int hwidth = layer->getWidth() / 2;
-  bool useCenterLine = false;
-  if (isMacroCellPin) {
-    auto rectDir = gtl::guess_orientation(rect);
-    if ((rectDir == gtl::HORIZONTAL && isLayer1Horz)
-        || (rectDir == gtl::VERTICAL && !isLayer1Horz)) {
-      auto layerWidth = layer->getWidth();
-      if ((rectDir == gtl::HORIZONTAL
-           && gtl::delta(rect, gtl::VERTICAL) < 2 * layerWidth)
-          || (rectDir == gtl::VERTICAL
-              && gtl::delta(rect, gtl::HORIZONTAL) < 2 * layerWidth)) {
-        useCenterLine = true;
+  bool use_center_line = false;
+  if (is_macro_cell_pin) {
+    auto rect_dir = gtl::guess_orientation(rect);
+    if ((rect_dir == gtl::HORIZONTAL && is_layer1_horz)
```

Comment:
I feel like this would benefit from a helper that said `LayerMatchesRectOrentation(layer, rect)`

Reply:
I'm noting this for the subsequent PR. For this one as it became pretty big do not intend to create new functions